lms/services/digest/_models.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass
+from functools import cached_property
+from typing import List
+
+from lms.services.mailchimp import EmailRecipient
+
+
+@dataclass
+class HUser(EmailRecipient):
+    """A class representing H users."""
+
+    h_userid: str
+    """The H userid like 'acct:name@lms.example.com'."""
+
+
+@dataclass
+class HCourse:
+    """A class representing courses as groups in H."""
+
+    title: str
+    """Title for the course."""
+
+    authority_provided_id: str
+    """The course's authority provided id."""
+
+    aka: List[str]
+    """All authority provided ids which map to the course from groups etc."""
+
+    instructors: List[str]
+    """A list of H userids for users who are instructors for the course."""
+
+    annotations: List[dict] = None
```

Comment:
If we were payed by number of data classes I'd make this one. Is hard to spot the the shape of these without reading multiple bits.

h_api.get_annotations could return that direclty.